scanner/src/main/kotlin/PostgresStorage.kt

```diff
@@ -184,11 +186,13 @@ class PostgresStorage(val connection: Connection) : ScanResultsStorage() {
             return false
         }
 
+        log.info { "Storing scan result for ${id.toCoordinates()} in storage." }
+
         // TODO: Check if there is already a matching entry for this provenance and scanner details.
         val query = "INSERT INTO $schema.$table (identifier, scan_result) VALUES (?, to_json(?::json)::jsonb)"
 
         val statement = connection.prepareStatement(query)
-        statement.setString(1, id.toString())
+        statement.setString(1, id.toCoordinates())
```

Comment:
Should this be squash to where the `toString()` call was introduced?

Reply:
Done.